src/components/MenuComponent.jsx

```diff
@@ -44,18 +45,26 @@ class Menu extends Component {
       text: 'Ausgaben',
       icon: <MoneyIcon />,
     },
+
+    {
+      id: 4,
+      link: '/logout',
+      text: 'Logout',
+      showName: true,
+      icon: <LogoutIcon />,
+    },
```

Comment:
Ich persönlich würde die Variable _menu_ in die Funktion _componentWillMount_ verschieben und dort den Text mit dem Benutzernamen ergänzen. Dann kann auf _showName_ verzichtet werden.
```suggestion
componentWillMount = () => {
    const { userName } = this.props;
    this.menu = [
      ...
      {
        id: 4,
        link: '/logout',
        text: `Logout ${userName}`,
        icon: <LogoutIcon />,
      },
    ];
  }
```

Reply:
Guter Hinweis, ist schöner so.